### DISCUSSION TITLE
I can't finalize the github developer pack request

### DISCUSSION THREAD

**Discussion by AristocratesJ** [community/community#110206] [GitHub Education] [answered]
### Hi everyone,

every time I try to finalize my request for students developer pack using "process my application" button I get redirected to same page but without my photo uploaded. I tried using different browsers in incognito mode, different types of uploaded file(.png, .jpg) and even tested using my phone to upload a request but nothing changed - the result is that I'm always getting redirected without any response. What else can i do? Also I can't select what type of proof I'm sending but i don't know if that's the problem or just site got changed.

**Comment by ThisIsMrIsmail** [comment]
Hello @AristocratesJ,
That sounds super annoying! Let's see if we can figure this out.

You could try the following steps:
1. **Check those document requirements:** Make sure your file is exactly what they're asking for.
2. **Wait a day (if you can):** Sometimes these things fix themselves. If not, move on to step 3.
3. **Hit up GitHub support:** If none of this works, you'll need to contact them: [https://support.github.com/contact](https://support.github.com/contact). Explain what's going on.

Don't give up! We'll get this figured out. 😊

**Comment by AristocratesJ** [comment, marked as answer]
Update:
eventually after checking browser console it turned out that site couldn't procces the photo and it threw "too large" error. It's strange because my photo was actually 900KB (so it met the requirements of size 100KB - 1MB). Idk, if requirements of the photo are misleading or it occured only in my case.
